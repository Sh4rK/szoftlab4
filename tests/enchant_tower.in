loadMap test.map
loadMission test.mission
buildTower 5 5
enchant 5 5 red
listTower
exit
