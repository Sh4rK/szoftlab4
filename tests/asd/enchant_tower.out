1 5 5 red
